designs/manager_policy_mapping/policy_mapping.md

```diff
@@ -0,0 +1,77 @@
+# Policy Mapping Workflow with IP/MAC
+
+This document describes the proposed changes for policy mapping workflow with IPs/MACs.
+
+## Background
+
+Currently, we have two ways to determine which policy gets applied to which workload:
+
+- Provide the VNF security manager with encapsulation tag
+- Provide the VNF security manager with IP/MAC
+
+If the manager is supporting [Policy Mapping](https://github.com/opensecuritycontroller/security-mgr-api/blob/master/src/main/java/org/osc/sdk/manager/api/ApplianceManagerApi.java#L40) and [Security Group](https://github.com/opensecuritycontroller/security-mgr-api/blob/master/src/main/java/org/osc/sdk/manager/api/ApplianceManagerApi.java#L41), we provide the manager with the security group name and members (their IPs and MACs) information to bind the policy.
```

Comment:
Use the links to the Const file instead, i.e.: https://github.com/opensecuritycontroller/security-mgr-api/blob/master/src/main/java/org/osc/sdk/manager/Constants.java#L70